tests/foreman/api/test_permission.py

```diff
@@ -193,19 +192,14 @@ def _permission_name(entity, which_perm):
 
 
 # This class might better belong in module test_multiple_paths.
-class UserRoleTestCase(APITestCase):
+class TestUserRole:
     """Give a user various permissions and see if they are enforced."""
 
-    @classmethod
-    def setUpClass(cls):
-        """Create common entities"""
-        super().setUpClass()
-        cls.org = entities.Organization().create()
-        cls.loc = entities.Location().create()
-
-    def setUp(self):  # noqa
+    @pytest.fixture(autouse=True)
+    def create_user(self):  # noqa
         """Create a set of credentials and a user."""
-        super().setUp()
+        self.org = entities.Organization().create()
```

Comment:
Functionally I think you're okay here - but note that this is now creating an organization and location at a `'function'` scope, instead of once for the whole class.

You can combine the fixture scopes here if you want to retain the behavior of unittest, creating organization and location entities in a class scoped fixture function, and then including that fixture name as a parameter to a function  scoped `'create_user'` fixture.